### ISSUE TITLE
Accessibility: MAS40B: Inspect: Properties are inappropriate for the tree items in "R Tools - Options" window

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/RTVS#3340]
**Platform:**
Windows 10 RS1
Visual Studio Enterprise 2017 Preview
Version: 15.0.26340.0

**Repro Steps:**
1. Launch Visual Studio
2. Hit "Alt+R", to open "Rtools" menu
3. Using arrow keys navigate to "Options" menu item and hit enter
4. Using tab and arrow keys navigate to "Debugging", "General", "Help", "History", "HTML", "Logging", "Markdown", "R Engine" and "Workspace" tree items and observe the properties in inspect

**Expected Result:**
Control type and localized control type should be defined. Ex. Tree Item 
IsExpandCollapsePatternAvailable property should be set to true

**Actual Result:**
Control type and localized control type are not supported
IsExpandCollapsePatternAvailable property is false

**Narrator Behavior:**
Narrator just announces as "Debugging". On hitting right/left arrow to expand/collapse narrator is announcing on right/left arrow

**User Impact:**
Screen reader dependent users will not know what is the control type currently focused hence cannot perform required action on the element. Also since IsExpandCollapsePatternAvailable is set to false screen reader users will not get to know about the state of the control whether expanded/collapsed

![inspect_tree items](https://cloud.githubusercontent.com/assets/22956130/24295570/5b6f5a2a-10c1-11e7-9bfa-7a4aae95a804.jpg)
![narrator log_tree items](https://cloud.githubusercontent.com/assets/22956130/24295571/5b6fc1f4-10c1-11e7-9cf8-a25c7456a40f.jpg)

**Comment by MikhailArkhipov** [comment]
Standard IDE grid behavior. Opened VSTS 399988
